examples/playbooks/contains_secrets.transformed.yml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
[This](https://github.com/cognifloyd/ansible-lint/blob/yaml-fmt/examples/playbooks/contains_secrets.transformed.yml) is a reformatted copy of [examples/playbooks/contains_secrets.yml](https://github.com/cognifloyd/ansible-lint/blob/yaml-fmt/examples/playbooks/contains_secrets.yml)